huggle/mainwindow.cpp

```diff
@@ -1501,13 +1501,13 @@ void MainWindow::OnTimerTick0()
 void MainWindow::on_actionNext_triggered()
 {
     if (!this->Queue1->Next())
-        this->ShowCat();
```

Comment:
If you really need to rename it, I think ShowEmptyQueuePage would make more sense, this function sounds like it sets a literally empty page, which is confusing